GitHub_Learning/Feature/Onboarding/View/OnboardingViewController.swift

```diff
@@ -12,13 +12,18 @@ import CleanPlatform
 
 class OnboardingViewController: BaseViewController {
     
-    let layout = OnboardingLayout()
-
+    private let layout = OnboardingLayout()
+    private let presenter: OnboardingPresenter
+    private var onboardingPagesVM = [OnboardingPageViewModel]()
```

Comment:
Prejmenoval bych to jednoduse na `pages`. `onboardingPagesVM` je zbytecne dlouhy.